webui/react/src/components/Logo.tsx

```diff
@@ -66,13 +67,15 @@ const Logo: React.FC<Props> = ({
     ].join();
   }, [branding]);
 
-  return (
-    <img
-      alt={alt}
-      className={classes.join(' ')}
-      src={logos[branding][orientation][isDarkMode ? 'dark' : 'light']}
-    />
-  );
+  const mode = isDarkMode ? 'dark' : 'light';
+  let imageSrc = logos[branding][orientation][mode];
+  let classes = [css[branding], css[orientation]];
+  if (hasCustomLogo) {
+    imageSrc = linkPath(paths.customerAsset('logo', orientation, mode), false);
+    classes = [css.customLogo];
```

Comment:
if conditional class needed, push instead of overwrite

Reply:
hmm I also want to make sure those other CSS rules associated with the other branding and oritentations don't apply to the user provided logos as they're very tailored to our specific logos. 
https://github.com/determined-ai/determined/blob/3baece2ff30bc4cf3ec1f54bfce63902791882b0/webui/react/src/components/Logo.module.scss#L1-L17 and for that I think it'd make sense to overwrite?

Reply:
actually looking at this more closely i don't think we even need the branding/orientation css. images should display based on their own dimensions anyway. I think a `base` class with `max-width` should be enough. and then the only thing that needs to be based on the `hasCustomLogo` condition is the `src`

Reply:
I can't quite tell why those were added originally so I feel a bit unsure about removing them wdy think if we minimize the changes to other existing logos here? if that makes sense I can put this up after to do that https://github.com/determined-ai/determined/pull/9692